packages/compiler-vapor/src/transforms/vBind.ts

```diff
@@ -21,6 +22,19 @@ export const transformVBind: DirectiveTransform = (dir, node, context) => {
     exp.ast = null
   }
 
+  if (modifiers.includes('camel')) {
+    if (arg.type === NodeTypes.SIMPLE_EXPRESSION) {
```

Comment:
`arg.type` is always a simple expression. Compound expression is not existing in Vapor compiler.